crates/topos-tce-storage/README.md

```diff
@@ -0,0 +1,81 @@
+# topos-tce-storage
+
+The library provides the storage layer for the Topos TCE.
+It is responsible for storing and retrieving the certificates, managing the
+pending certificates pool and the certificate status, storing the different
+metadata related to the protocol and the internal state of the TCE.
+
+The storage layer is implemented using RocksDB.
+The library is exposing multiple store that are used by the TCE.
+
+
+### Architecture
+
+The storage layer is composed of multiple stores that are used by the TCE.
+Each store is describe in detail in its own module.
+
+As an overview, the storage layer is composed of the following stores:
+
+<picture>
```

Comment:
Picture is a bit confusing `ValidatorPerEpochStore` and `EpochValidatorStore`? 
`ValidatorPerEpochStore` is not mentioned in the text

Reply:
It is not use for now as we only have on active `epoch` yes